demisto_sdk/commands/common/content/objects/pack_objects/abstract_pack_objects/json_content_object.py

```diff
@@ -76,7 +76,7 @@ def readme(self) -> Optional[Readme]:
         return self._readme
 
     @property
-    def from_version(self) -> Union[Version, LegacyVersion]:
+    def from_version(self) -> Union[Version]:
```

Comment:
Please remove the Union
```suggestion
    def from_version(self) -> Version:
```